fee_calculator/apps/calculator/tests/data/test_dataset_lgfs_2016.csv

```diff
@@ -6040,3 +6040,5 @@ CASE_ID,BILL_TYPE,BILL_SUB_TYPE,REP_ORD_DATE,CALC_FEE_EXC_VAT,CALC_FEE_VAT,ACTUA
 1076111,LIT_FEE,LIT_FEE,01/09/2016,177.29,35.46,177.29,35.46,B,261,0,15,1,ST1TS0T9
 1076140,LIT_FEE,LIT_FEE,28/04/2016,709.15,141.83,709.15,141.83,B,268,0,26,1,ST3TS2T3
 1076166,LIT_FEE,LIT_FEE,30/06/2016,709.15,141.83,709.15,141.83,B,268,0,47,1,ST3TS2T3
+X065110,LIT_FEE,LIT_FEE,17/08/2016,0,0,0,0,K,293,8,27391,1,ST1TS3T0
```

Comment:
does the `X` matter?

Reply:
I just put the X in to denote that this is a fake test line, and doesn't match up to any of the cases in CCLF.